src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/HandlerFactory.java

```diff
@@ -74,6 +74,8 @@ public HandlerFactory() {
         register(TokenTypes.INDEX_OP, IndexHandler.class);
         register(TokenTypes.LITERAL_SYNCHRONIZED, SynchronizedHandler.class);
         register(TokenTypes.LAMBDA, LambdaHandler.class);
+        register(TokenTypes.ANNOTATION_DEF, ClassDefHandler.class);
+        register(TokenTypes.ANNOTATION_FIELD_DEF, AnnotationFieldDefHandler.class);
```

Comment:
I still didn't receive an answer or change from my previous comment. https://github.com/checkstyle/checkstyle/pull/4217#discussion_r111636791

> > AnnotationDefHandler looks like it is coded like MemberDefHandler.
We can't use that handler, or give them common parents, instead? Does it make sense to do this?

> Yes I based the field checking on member def handler.

That makes it sound more like they should have comment class hierarchy.